app/src/main/java/dev/arkbuilders/arkretouch/storage/ImageDefaults.kt

```diff
@@ -1,9 +1,10 @@
-package dev.arkbuilders.arkretouch.data
+package dev.arkbuilders.arkretouch.storage
 
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.IntSize
 import kotlinx.serialization.Serializable
 
+// FIXME: Replace with Parcelable
```

Comment:
Is it worth of creating follow-up issue for this? Or can be done in this PR?

Reply:
We need issue since it can lead to crashes

Reply:
- #95